### ISSUE TITLE
Default colours need adjusting!

### ISSUE THREAD

**Issue by davidc** [petere/emacs-eruby-mode#2]
White on white isn't very visible.

![image](https://user-images.githubusercontent.com/1050152/37361993-6e7d145a-26f4-11e8-826d-4688798f1ee6.png)

This may be better:

```
--- eruby-mode.el.orig  2018-03-13 18:26:45.244487806 +0000
+++ eruby-mode.el       2018-03-13 18:27:02.896714954 +0000
@@ -46,7 +46,7 @@
   :group 'faces)

 (defface eruby-standard-face
-  '((t (:background "gray")))
+  '((t (:foreground "yellow")))
   "Face used to highlight eRuby template snippets"
   :group 'eruby-mode-faces)

```

**Comment by petere** [comment]
I think any default choice is going to conflict with someone's window or terminal settings. Just changing one color to another isn't going to fundamentally fix that.

**Comment by davidc** [comment]
True, but those people have already customised their terminals and should expect such things. Whereas I've never seen a terminal that isn't by default white on black or black on white, so using either of these colours by default seems like a bad idea.

**Comment by petere** [comment]
My terminal is black on white. Gray is neither black nor white, so it shows up well. Yellow on the other hand is barely visible against the white. So mileage does vary.
